src/main/java/io/confluent/connect/hdfs/DataWriter.java

```diff
@@ -109,19 +109,23 @@ public DataWriter(HdfsSinkConnectorConfig connectorConfig, SinkTaskContext conte
               + "the path to the keytab of the principal.");
         }
 
+        conf.set("hadoop.security.authentication", "kerberos");
+        conf.set("hadoop.security.authorization", "true");
```

Comment:
This makes sure that the connector uses kerberos for authentication even if the use didn’t provide the hadoop.conf.dir in the connector configuration